django/template/defaultfilters.py

```diff
@@ -970,9 +974,17 @@ def phone2numeric_filter(value):
 
 
 @register.filter(is_safe=True)
-def pprint(value):
-    """A wrapper around pprint.pprint -- for debugging, really."""
+def pprint(v):
```

Comment:
I would leave `value` unchanged here and below for readability, and for a similar reason the docstring.
```suggestion
def pprint(value):
```

Reply:
Gotcha, didn't notice that. 👍

Reply:
Please rename also other occurrences of `v`

Reply:
I have done so locally, will push with other changes 👍